test/core/core_texture.cpp

```diff
@@ -6,22 +10,16 @@
 #include <gli/load.hpp>
 #include <gli/copy.hpp>
 #include <gli/generate_mipmaps.hpp>
+
 #include <glm/gtc/epsilon.hpp>
-#include <ctime>
-#include <memory>
 
 namespace alloc
 {
 	int run()
 	{
-		int Error(0);
+		int Error = 0;
 
-		std::vector<int> Sizes;
-		Sizes.push_back(16);
-		Sizes.push_back(32);
-		Sizes.push_back(15);
-		Sizes.push_back(17);
-		Sizes.push_back(1);
+		std::vector<int> Sizes = { 16, 32, 15, 17, 1 };
```

Comment:
Debatable whether changes like these are needed. Theoretically, initializing the vector only once avoids the needless bookkeeping on `push_back` calls.